Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/Products'...
Listing './lib/python3.5/site-packages/Products/PloneArticle'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/Extensions'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/browser'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/browser/article.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/browser/article.py", line 104
    raise ValueError, "Proxy must implements IImageInnerContentProxy"
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/Products/PloneArticle/browser/explorer.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/browser/explorer.py", line 50
    raise ValueError, ("%s called without 'current_article' in REQUEST",
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/Products/PloneArticle/browser/multipage.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/browser/multipage.py", line 54
    raise ValueError, "not in a multipage article"
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/browser/templates'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/content'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/doc'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/doc/useCases'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel/Extensions'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel/browser'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel/i18n'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel/skins'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/examples/mypamodel/skins/mypamodel'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/field'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/field/baseinnercontent.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/field/baseinnercontent.py", line 86
    except Exception, msg:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/Products/PloneArticle/field/innercontentcontainer.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/field/innercontentcontainer.py", line 87
    raise ValueError, 'Disallowed subobject type: %s' % type_name
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/Products/PloneArticle/field/smartlist.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/field/smartlist.py", line 117
    raise TypeError, "Invalid type for search_criterias"
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/ftests/framework.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/ftests/framework.py", line 88
    print 'Unable to locate Testing package.',
                                            ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/data'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/logical'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/physical'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/scenarios'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/scripts'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/ftests/plonearticle/tests'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/i18n'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/interfaces'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/migration'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/migration/v4'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/migration/v4/betas.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/migration/v4/betas.py", line 104
    except AttributeError, e:
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/migration/v4_1'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/model'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/model/__init__.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/model/__init__.py", line 31
    raise ValueError, \
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/profiles'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/profiles/default'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/profiles/default/types'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/proxy'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/setuphandlers.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/setuphandlers.py", line 124
    print 'contenttype removed from kupu resource %s : %s \n' %(resource_type, type_id )
                                                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/skins'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/getBrowserCrumbs.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/getBrowserCrumbs.py", line 36
    return result
                ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/getRelPath.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/getRelPath.py", line 17
    return relPath
                 ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_create_inner_content_widget.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_create_inner_content_widget.py", line 22
    return template(field_name=field_name, field=field, widget=widget,
                                                                     ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innercontent_reference.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innercontent_reference.py", line 18
    return context.standard_error_message(error_type=404,
    ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innercontent_upload.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innercontent_upload.py", line 42
    return template(from_upload=True, skip_widget=True, fail_reason=message)
    ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerfile_reference.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerfile_reference.py", line 11
    return context.pa_innercontent_reference(field_name, uid, widget_index)
                                                                          ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerfile_upload.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerfile_upload.py", line 11
    return context.pa_innercontent_upload(
                                         ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerimage_reference.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerimage_reference.py", line 11
    return context.pa_innercontent_reference(field_name, uid, widget_index)
                                                                          ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerimage_upload.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerimage_upload.py", line 11
    return context.pa_innercontent_upload(
                                         ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerlink_reference.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerlink_reference.py", line 11
    return context.pa_innercontent_reference(field_name, uid, widget_index)
                                                                          ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerlink_upload.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_innerlink_upload.py", line 11
    return context.pa_innercontent_upload(
                                         ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_thumb.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle/pa_thumb.py", line 17
    raise NotFound, "Unknown page."
                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle_images'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/skins/plonearticle_models'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/tests'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/tests/input'...
Compiling './lib/python3.5/site-packages/Products/PloneArticle/tool.py'...
***   File "./lib/python3.5/site-packages/Products/PloneArticle/tool.py", line 335
    raise ValueError, 'Scale can only be applied on Image'
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/Products/PloneArticle/widget'...
Listing './lib/python3.5/site-packages/Products/PloneArticle/www'...
Listing './lib/python3.5/site-packages/Products.PloneArticle-4.2.0.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
